### PR TITLE
Ship V6 to prod

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**
preview here: https://documentations.iex.ec/staging/v/v6-staging/

### CODE REVIEW

**Comment by PierreJeanjacquot** [comment]
preview here: https://documentations.iex.ec/staging/v/v6-staging/

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 1]
```diff
@@ -15,199 +15,204 @@ Please make sure you have already checked the [quickstart](../your-first-app.md)
...
 
-{% tab title="Python" %}
-{% code title="Dockefile" %}
-```bash
-FROM sconecuratedimages/public-apps:python-3.7.3-alpine3.10-scone3.0
+## Build the TEE docker image:
 
-### install python3 dependencies you need
-RUN SCONE_MODE=sim pip3 install pyfiglet
+You will need to register a [free SCONE Account](https://scontain.com) to access the SCONE build tools and curated images from the [SCONE registry](https://gitlab.scontain.com/).
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]:
```suggestion
You will need to register a [free SCONE Account](https://scontain.com) to access SCONE build tools and curated images from the [SCONE registry](https://gitlab.scontain.com/).
```

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 2]
```diff
@@ -15,199 +15,204 @@ Please make sure you have already checked the [quickstart](../your-first-app.md)
...
 
 {% hint style="info" %}
-All dependencies and files must be added to the image before invoking the **protect-fs.sh** script \(see below\).
-{% endhint %}
 
-{% hint style="warning" %}
-It is important to carefully choose files to authenticate. It can be tricky to consider including enough files to protect the application without being more general than we should. For instance, if we authenticate the entire **/etc** directory the enclave will fail to boot because the content of /etc/hosts is modified at runtime by Docker.
-{% endhint %}
+You may have noticed the `tee-debug` flag in the image name, the built image is actually in TEE debug mode, this allows you to have some debug features while developping the app.
+Once you are happy with the debug app, contact-us to go to production!
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]:
```suggestion
Once you are happy with the debug app, contact us to go to production!
```

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: @jeremyjams direct commit on staging is blocked, we have to open a new PR `v6-fixes` -> `v6-staging`

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: same

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 1 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: fixed f04c7e2704c82b5ff28637f07499c3681283e838

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 3]
```diff
@@ -15,199 +15,204 @@ Please make sure you have already checked the [quickstart](../your-first-app.md)
...
 }
-#####################################################################
-Hint: Replace 'mrenclave' before doing 'iexec app deploy' step.
 ```
 
-That alphanumeric string is the [fingerprint](intel-sgx-technology.md#applications-fingerprint) of your application. It allows the verification of its integrity.
+{% hint style="info" %}
+Run your TEE image with `SCONE_HASH=1` to get the enclave fingerprint (mrenclave):
+```sh
+docker run -it --rm -e SCONE_HASH=1 nodejs-hello-world-with-dataset:tee-debug
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]:
```suggestion
docker run -it --rm -e SCONE_HASH=1 nodejs-hello-world:tee-debug
```

Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: fixed 4baa2bfcc0455dbd2d76374f3b5b46d7cf140a08

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md:130]: fixed b5d6055ce7bb395e6430d664119b067ae6ba18d1

Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: fixed b5d6055ce7bb395e6430d664119b067ae6ba18d1

File: for-developers/confidential-computing/sgx-encrypted-dataset.md [hunk 4]
```diff
@@ -126,22 +122,31 @@ $ cat iexec.json
...
-iexec dataset deploy --chain goerli
+iexec dataset deploy --chain viviani
 ```
 
 You will get a hexadecimal address for your deployed dataset. Use that address to push the encryption key to the [SMS](intel-sgx-technology.md#secret-management-service-sms) so it is available for authorized applications.
 
-```bash
-iexec dataset push-secret <0x-your-dataset-address> --chain goerli
-```
+For simplicity, we will use the dataset with a TEE-debug app on a debug workerpool. The debug workerpool is connected to a debug Secret Management System so we will send the dataset's encryption key to this SMS (this is fine for debuging but do not use to store production secrets).
```
Comment (james-toussaint) [inline for-developers/confidential-computing/sgx-encrypted-dataset.md]:
```suggestion
For simplicity, we will use the dataset with a TEE-debug app on a debug workerpool. The debug workerpool is connected to a debug Secret Management Service so we will send the dataset encryption key to this SMS (this is fine for debuging but do not use to store production secrets).
```

File: for-developers/confidential-computing/sgx-encrypted-dataset.md [hunk 5]
```diff
@@ -199,187 +206,131 @@ const figlet = require('figlet');
...
+# declare an image name
+IMG_NAME=nodejs-dataset-app
+
+IMG_FROM=${IMG_NAME}:temp-non-tee
+IMG_TO=${IMG_NAME}:tee-debug
+
+# build the regular non-TEE image
+docker build . -t ${IMG_FROM}
+
+# pull the SCONE currated image corresponding to our base image
```
Comment (james-toussaint) [inline for-developers/confidential-computing/sgx-encrypted-dataset.md]:
```suggestion
# pull the SCONE curated image corresponding to our base image
```

File: for-developers/confidential-computing/create-your-first-sgx-app.md [hunk 6]
```diff
@@ -15,199 +15,204 @@ Please make sure you have already checked the [quickstart](../your-first-app.md)
...
+# declare an image name
+IMG_NAME=nodejs-hello-world
+
+IMG_FROM=${IMG_NAME}:temp-non-tee
+IMG_TO=${IMG_NAME}:tee-debug
+
+# build the regular non-TEE image
+docker build . -t ${IMG_FROM}
+
+# pull the SCONE currated image corresponding to our base image
```
Comment (james-toussaint) [inline for-developers/confidential-computing/create-your-first-sgx-app.md]:
```suggestion
# pull the SCONE curated image corresponding to our base image
```

**Review by james-toussaint — APPROVED** [review]

File: for-developers/confidential-computing/sgx-encrypted-dataset.md (hunk 4 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/sgx-encrypted-dataset.md]: fixed 83a1f2ba28354c09ad4466f9d43faca9ebc1e390

File: for-developers/confidential-computing/sgx-encrypted-dataset.md (hunk 5 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/sgx-encrypted-dataset.md]: fixed 5aba1103bccfd7bf2338daa13660a9b7cf17a91b

File: for-developers/confidential-computing/create-your-first-sgx-app.md (hunk 6 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/create-your-first-sgx-app.md]: fixed c6ef64bcf57683fcfb57134120a47f44336acc63

File: for-developers/confidential-computing/sgx-encrypted-dataset.md [hunk 7]
```diff
@@ -127,7 +127,7 @@ iexec dataset deploy --chain viviani
 
 You will get a hexadecimal address for your deployed dataset. Use that address to push the encryption key to the [SMS](intel-sgx-technology.md#secret-management-service-sms) so it is available for authorized applications.
 
-For simplicity, we will use the dataset with a TEE-debug app on a debug workerpool. The debug workerpool is connected to a debug Secret Management System so we will send the dataset's encryption key to this SMS (this is fine for debuging but do not use to store production secrets).
+For simplicity, we will use the dataset with a TEE-debug app on a debug workerpool. The debug workerpool is connected to a debug Secret Management Service so we will send the dataset encryption key to this SMS (this is fine for debuging but do not use to store production secrets).
```
Comment (ericr6) [inline for-developers/confidential-computing/sgx-encrypted-dataset.md:130]: debugging

**Review by ericr6 — APPROVED** [review]

File: for-developers/confidential-computing/sgx-encrypted-dataset.md (hunk 7 above)
Comment (PierreJeanjacquot) [reply for-developers/confidential-computing/sgx-encrypted-dataset.md:130]: fixed 1155ad4a3479dc0b85c4184f7da62602e833c5e6

**Pull request merged**